Modules/Core/Common/include/itkNeighborhoodOperator.h

```diff
@@ -87,6 +87,12 @@ class ITK_TEMPLATE_EXPORT NeighborhoodOperator : public Neighborhood<TPixel, VDi
   /** Slice iterator type alias support */
   using SliceIteratorType = SliceIterator<TPixel, Self>;
 
+  using PixelRealType = typename NumericTraits<TPixel>::RealType;
+
+  /** Typedef support for coefficient vector type. Necessary
+   * to fix bug in the microsoft VC++ compiler. */
+  using CoefficientVector = std::vector<typename PixelRealType>;
```

Comment:
Is CoefficientVector needed to be public? It was protected before.

Reply:
You are right, but `PixelRealtype` was/is defined as public, and thus I moved it to the public section. And `PixelRealType` seems to be required in the public section so that the public method `ScaleCoefficients` is allowed to use it.

Reply:
Finally, as it seems that the `CoefficientVector` is not being inherited I put it back to the protected section.